lib/msal-common/src/authority/Authority.ts

```diff
@@ -552,17 +552,26 @@ export class Authority {
                 : [];
             if (metadata.length === 0) {
                 // If no metadata is returned, authority is untrusted
+                this.logger.warning(`The cloud instance discovery network request's status code is: ${response.status}`);
+                this.logger.warning("There weren't any cloud instances returned.");
+                
+                const tenantDiscoveryEnpoint = response.body.tenant_discovery_endpoint;
+                const tenantDiscoveryEnpointLoggerHelper = tenantDiscoveryEnpoint.length ? `: ${tenantDiscoveryEnpoint.length}` : " an empty string";
+                this.logger.warningPii(`tenant_discovery_endpoint is${tenantDiscoveryEnpointLoggerHelper}`);
                 return null;
             }
             match = Authority.getCloudDiscoveryMetadataFromNetworkResponse(
                 metadata,
                 this.hostnameAndPort
             );
         } catch (e) {
+            this.logger.error(`There was a network error while attempting to get the cloud discovery instance metadata: ${e}`);
```

Comment:
nit: not necessarily a network error. Could be because of a null pointer ex somewhere above.
```suggestion
            this.logger.error(`There was an error while attempting to get the cloud discovery instance metadata: ${e}`);
```